### PR TITLE
fix: scrollbar pushing UI to the left on appearance

### LINKED ISSUE #1644: Scrollbar pushing Ui to the left on appearance
Feb-04-2022 17-51-25

Possible fix: https://stackoverflow.com/questions/35513264/pure-css-way-to-prevent-scrollbar-pushing-content-to-the-left

### PR DESCRIPTION
**Pull request by bigint**
Fixes #1644

https://user-images.githubusercontent.com/69431456/152534914-ad50e9fe-461e-4506-9e9c-82e083e5128b.mov

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
 @media (pointer: fine) {
   body {
-    overscroll-behavior-y: none;
+    overflow: overlay;
```
Comment (mktcode) [inline src/style.scss:80]: Why did you remove `overscroll-behavior-y`? Also `overflow: overlay` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values) and won't work in Firefox.

**Comment by samuveth** [comment]
https://github.com/snapshot-labs/snapshot/pull/1686
